packages/elements/package.json

```diff
@@ -95,7 +95,7 @@
   "peerDependencies": {
     "@clerk/clerk-react": "^5.0.0",
     "@clerk/shared": "^2.0.0",
-    "next": "^13.5.4 || ^14.0.3",
+    "next": "^13.5.4 || ^14.0.3 || >=15.0.0-rc",
     "react": ">=18",
     "react-dom": ">=18"
```

Comment:
@LekoArts shall we also do `react@rc` and `react-dom@rc` ?

Reply:
Yeah, good callout 👍